hw/dv/tools/xcelium/common_cov_excl.tcl

```diff
@@ -0,0 +1,8 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+exclude -inst $::env(DUT_TOP) -toggle '*tl_i.a_user.rsvd' -comment "\[UNSUPPORTED\] Based on our Comportability Spec. Exercising this will result in assertion errors thrown."
```

Comment:
I would like to mandate that all exclusions have a comment, and those comments be prefixed with a known tag so that it is easier to grep. These tags are documented in our DV methodology doc.